mri2mrs.py

```diff
@@ -0,0 +1,218 @@
+            fslmaths[convertedImg, '-mul', bet_mask, processedImg] & FG
+
+        else:
+            processedImg= convertedImg
+
+        warpDim(processedImg)
+
+        log('Segmenting T1 image to white/gray/CSF ...')
+        ((fast['-o', 'fast_out', processedImg] ) >> logFile)()
+
+
+        log('Creating white, gray, csf separate images ...')
+
+        white = 'fast_out_seg.nii.gz'
+        gray = 'fast_out_seg2.nii.gz'
+        csf = 'fast_out_seg3.nii.gz'
+
+
+        fslmaths[white, '-uthr', '1', gray] & FG
+        fslmaths[gray, '-add', '1', gray] & FG
+        fslmaths[gray, '-uthr', '1', gray] & FG
+
+        fslmaths[white, '-thr', '3', csf] & FG
+        fslmaths[csf, '-div', '3', csf] & FG
+
+
+        # Ofer's MATLAB code reads only .nhdr file
+        compatibleImg= self.case+'-t1w.nhdr'
+        # TODO: Check with Ofer if he wants unprocessed image as the first argument
+        ((ConvertBetweenFileFormats[processedImg, compatibleImg] ) >> logFile)()
+
+        try:
+            regionPrefix = self.region
+            log('Defining MRS on T1 image using MATLAB ...')
+
+            retcode, stdout, stderr= matlab.run(['-singleCompThread', '-nojvm', '-nosplash', '-r',
+                   "addpath {}, MRStoAnatomy(\'{}\', \'{}\', \'{}\', \'{}_mask\'), exit"
+                   .format(scriptDir, 'tmp-sb.nhdr', compatibleImg, self.labelMap, regionPrefix)])
+            externalOutput(retcode, stdout, stderr)
+
+
+            log('Combining the mask with the zero files to create a mask having same dimensions as that of T1, '
+                'launching Slicer ...')
+
+            retcode, stdout, stderr= Slicer.run(['--launch', 'AddScalarVolumes',
+                   self.region+'_mask_zr.nhdr',
+                   self.region + '_mask_jm.nhdr',
+                   '--order', '0',
+                   self.region + '_mask.nhdr'])
+            externalOutput(retcode, stdout, stderr)
+
+
+        except:
+            log('MRS mask creation failed for {}, exiting ...'.format(self.img))
+            exit(1)
+
+        log("Converting MRS mask to nii ...")
+        oldMRSmask= regionPrefix+'_mask.nhdr'
+        newMRSmask= regionPrefix+'_mask.nii.gz'
+        ((ConvertBetweenFileFormats[oldMRSmask, newMRSmask] ) >> logFile)()
+
+        warpDim(newMRSmask)
+
+        log('Creating MRS masks ...')
+        MRSmaskBrain= regionPrefix+'_MRS_mask_brain.nii.gz'
+        MRSmaskWM = regionPrefix + '_MRS_mask_wm.nii.gz'
+        createMRSmask(gray, newMRSmask, MRSmaskBrain)
+        createMRSmask(csf, newMRSmask, MRSmaskWM)
+
+        log('Calculating brain volume ...')
+        log('Brain volume : {}'.format(calcVol(MRSmaskBrain)))
+
+        log('Calculating white matter volume ...')
+        log('White matter volume : {}'.format(calcVol(MRSmaskWM)))
+
+        print('Program finished, see {} for details'.format(logFile))
+
+        os.chdir(scriptDir)
+        f.close()
+
+if __name__ == '__main__':
+    MRS.run()
+
+
+'''
```

Comment:
Put this in the script help output?

Reply:
Yeah, sorry about that. I forgot to take them off. They are already in the README.md file